spec/classes/cron_spec.rb

```diff
@@ -127,10 +127,12 @@
         }
       end
 
+
       context 'manage_crontab => true' do
         let(:params) do
           {
-            manage_crontab: true
+            manage_crontab: true,
+            crontab_file_mode: '0600',
```

Comment:
crontab_file_mode should not be specified here, it should be specified in its own test. We need to test both how it handles when the parameter is not set and when it is set.